cmd/root_commands.go

```diff
@@ -131,3 +140,20 @@ func initCustomiseMyData(cfg *config.Config) *cobra.Command {
 		},
 	}
 }
+
+func generateRepository() *cobra.Command {
+	return &cobra.Command{
+		Use:   "github",
+		Short: "Creates a github hosted repository",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var err error
+
+			err = repocreation.GenerateGithubRepository()
```

Comment:
There is potential this tool could be used for another repo host such as bitbucket, so in the spirit of keeping the code base open for expansion, I will have the package called 'repository' as mentioned but then instead of `Generate` or `GenerateGithubRepository` call is `GenerateGithub` so it will be `repository.GenerateGithub`